wiki/Installation.md

```diff
@@ -1,36 +1,70 @@
 # Swift Package Manager
-If you want more information about installing Swift packages with Xcode, [follow these instructions](https://developer.apple.com/documentation/swift_packages/adding_package_dependencies_to_your_app). If you want to learn more about Swift Package Manager, [this page goes into detail](https://swift.org/package-manager/).
+
+If you want more information about installing Swift packages through Xcode, [follow these instructions](https://developer.apple.com/documentation/swift_packages/adding_package_dependencies_to_your_app). If you want to learn more about Swift Package Manager, [this page goes into detail](https://swift.org/package-manager/).
+
+If you're a framework author and want to use SwiftCurrent as a dependency, update your `Package.swift` file:
 
 ## Get the package
+
 Add the following line to the package dependencies in `Package.swift`:
+
 ```swift
-.package(url: "https://github.com/wwt/Workflow.git", .upToNextMajor(from: "3.0.0")),
+.package(url: "https://github.com/wwt/SwiftCurrent.git", .upToNextMajor(from: "3.0.0")),
 ```
 
 ## Get the correct product
+
 Add one one of the following products to your target dependencies.
-### If you want to use Workflow with UIKit:
+
+#### **If you want to use SwiftCurrent with UIKit**
+
 ```swift
-.product(name: "WorkflowUIKit", package: "Workflow")
+.product(name: "SwiftCurrent", package: "SwiftCurrent"),
+.product(name: "SwiftCurrent_UIKit", package: "SwiftCurrent_UIKit"")
 ```
+
+#### **Your import statement in this case will be**
+
+```swift
+import SwiftCurrent_UIKit
```

Comment:
Since it looks like we are saying that you also need `SwiftCurrent` we should include here with maybe a quick why?  I'm on the fence about this though.

Reply:
I can see that - did you have some verbiage in mind for the why?

Reply:
Nope.